tests/src/testnonlinearelasticity.hh

```diff
@@ -198,7 +199,7 @@ auto GreenLagrangeStrainTest(const Material& mat) {
   auto nDOF    = basis.flat().size();
 
   auto fe   = makeFE(basis, skills(nonLinearElastic(mat)));
-  auto feLE = makeFE(basis, skills(Ikarus::linearElastic({.emodul = 1000, .nu = 0.0})));
+  auto feLE = makeFE(basis, skills(Ikarus::linearElastic({.emodul = 1000, .nu = 0.3})));
```

Comment:
I think this does not make much since, can you extract youngs moduls and nu from the `mat? variable, then it will work forever. Because we also test wit `nu=0.0` this should also fail.

Reply:
I cannot find this line in your changes. Somehow I aw it is written `mat.getParameters()` but this looks like it was already there in the code and not as a contribution from your PR

Reply:
Yes this was never a contribution from the PR. The only thing that was changed was the Poisson's ratio to check if the linear and the nonlinear stiffness matrix are the same for non-zero Poisson's ratio as well. Anyways, the fix that you wanted was already done in #317

Reply:
Ah I see.